prow/pod-utils/decorate/podspec.go

```diff
@@ -221,97 +378,15 @@ func decorate(spec *kube.PodSpec, pj *kube.ProwJob, rawEnv map[string]string) er
 		},
 	}
 
-	var cloneVolumes []kube.Volume
-	var cloneLog string
-	var refs []*kube.Refs
-	if pj.Spec.Refs != nil {
-		refs = append(refs, pj.Spec.Refs)
+	// TODO(fejta): decorated containers should run before other init containers
```

Comment:
Not sure this is necessary as it's predicated on the concept that there will ever be any other init containers, which is not yet true